content/docs/miscellaneous/acronyms_and_abbreviations.md

```diff
@@ -0,0 +1,85 @@
+---
+# Page title as it appears in the navigation menu
+title: "Acronyms and Abbreviations"
+# Adjust weight to reorder menu items (lower numbers appear first)
+weight: 1
+# Uncomment to hide nested pages in a collapsed menu
+# bookCollapseSection = true
+# Uncomment to hide this page from the navigation menu
+# bookHidden = false
+# Uncomment to exclude this page from the search
+# bookSearchExclude = true
+---
+
+# Acronyms and Abbreviations
+
+AIDA: Artificial Intelligence for Data Analytics
+ALOGIT: name of a Fortran package (see [issue](https://github.com/alan-turing-institute/Hut23/issues/295))
+ASG: AI for Science and Government (the new name of the EPSRC funds from SPF)
+
+ARC: Australian Research Council
+
+BL: British Library
+BPT: Bayesian Performance Tuning
+
+Chrono: Chronotopic Cartographies
+CPD: Change-point detection (method to apply to data in AIDA-Lloyds project)
+CT: Computerised Tomography
+
+DAS: Data Availability Statements (Giovanni's 22-days research paper)
+DAFNI: [Data & Analytics Facility for National Infrastructure](https://www.dafni.ac.uk/)
+DC: Detector-Checker
+
+D&S: Defence and Security (Programme)
+DS&DH: Data Science & Digital Humanities (Special Interest Group)
+DSB: Data Science Benchmarking
+DS for SD: Data Science for Sustainable Development
+DSG: Data Study Group
+DSSG: Data Science for Social Good
+DSSH: Data Science for Science and the Humanities (Programme)
+
+EDS: Economic Data Science
+EOM: End of message
+
+GUARD: Global Urban Analytics for Resilient Defence
+
+HPC: High Performance Computing
+
+IAG: International Airlines Group (British Airways merged with Iberia to form IAG)
+
+KF: Knowledge Flows
+
+LwM: Living with Machines
+
+MRO: ?
```

Comment:
We shouldn't include this if we don't know what it is.

Reply:
Agreed